api/envoy/extensions/filters/http/header_mutation/v3/header_mutation.proto

```diff
@@ -21,6 +22,8 @@ message Mutations {
 
   // The response mutations are applied before the response is sent to the downstream client.
   repeated config.common.mutation_rules.v3.HeaderMutation response_mutations = 2;
+
+  repeated config.core.v3.KeyValueMutation query_parameter_mutations = 3;
```

Comment:
What is the order between the `query_parameter_mutations` and `request_mutations`? Please document it in the comment of the API.